src/android/HappieCameraActivity.java

```diff
@@ -196,6 +196,15 @@ protected void initCameraSession() {
                     params.setFlashMode(Camera.Parameters.FLASH_MODE_TORCH);
                     flash.setImageResource(R.drawable.camera_flash_on);
             }
+            if(params.getSupportedPictureSizes()!=null) {
+                Camera.Size currentSize = params.getPictureSize();
+                Camera.Size maxSize = params.getSupportedPictureSizes().get(0);
+                if(currentSize.height<maxSize.height ||
+                        currentSize.width < maxSize.width) {
+                    params.setPictureSize(maxSize.width, maxSize.height);
+                }
+            }
+            params.setJpegQuality(100);
```

Comment:
@rishikapadia1 thanks! is the resolution affected by setting setJpegQuality(85)? From from tests I did a while ago, 85 reduces the file size without noticeably reducing image quality.